packages/components/src/components/tabs/tabs.jsx

```diff
@@ -34,12 +34,18 @@ class Tabs extends React.Component {
         if (this.props.header_fit_content && this.active_tab_ref && this.tabs_wrapper_ref) {
             const tabs_wrapper_bounds = this.tabs_wrapper_ref.getBoundingClientRect();
             const active_tab_bounds = this.active_tab_ref.getBoundingClientRect();
-            this.setState({
-                active_line_style: {
-                    left: active_tab_bounds.left - tabs_wrapper_bounds.left,
-                    width: active_tab_bounds.width,
-                },
-            });
+            if (active_tab_bounds.width === 0) {
+                setTimeout(() => {
+                    this.setActiveLineStyle();
```

Comment:
any chance this will be in an infinite loop? 🤔

Reply:
No. A few lines above, there is an `if` so we're sure the element actually exists but we need some time for component to capture the actual height.